packages/node/src/Client.ts

```diff
@@ -50,7 +50,7 @@ class Client {
   }
 
   deliver(parcel: Parcel): Promise<Response> {
-    return this.makeHttpRequest('/deliver', parcel)
+    return this.makeHttpRequest('deliver', parcel)
```

Comment:
Can we have the slash here in all the `this.makeHttpRequest('/xyz')` calls?

Reply:
We can, I omitted it because I felt it's redundant

Reply:
If we are adding `/` I'll have to create another PR for that.